libgnn/test/accuracy-test.cpp

```diff
@@ -5,6 +5,8 @@
 #include "galois/Logging.h"
 #include "galois/GraphNeuralNetwork.h"
 
+#define TESTER_DIR TESTINPUT "/tester/"
```

Comment:
if you override this as `${SRC_DIR}/inputs` then the next subpath `tester` is lost so the paths are messed up

I would recommend using definitions for both cora and tester or change the tests to add that subpath themselves

Reply:
Thank you!